test/cppsuite/tests/poc.cxx

```diff
@@ -88,6 +87,12 @@ main(int argc, char *argv[])
     std::cout << "Configuration\t:" << cfg << std::endl;
     std::cout << "Trace level\t:" << trace_level << std::endl;
 
-    poc_test(cfg, trace_level).run();
+    try {
+        poc_test(cfg, trace_level).run();
+    } catch (const std::exception &e) {
```

Comment:
Instead of catching an exception here use testutil_die with a message so we're consistent.